src/globus_cli/commands/update.py

```diff
@@ -48,19 +58,11 @@ def _check_pip_installed():
     disable_options=["format", "map_http_status"],
     short_help="Update the Globus CLI to its  latest version",
 )
+@click.option("--force", is_flag=True, hidden=True)
 @click.option("--yes", is_flag=True, help='Automatically say "yes" to all prompts')
-# hidden options to fetch branches or tags from GitHub. One turns this mode
-# on or off, and the other is used to set a non-main target
-# --development-version implies --development
-@click.option("--development", is_flag=True, hidden=True)
-@click.option("--development-version", hidden=True, default=None)
-def update_command(yes, development, development_version):
+def update_command(yes, force, development, development_version):
```

Comment:
Probably should remove the development and development_version arguments here as well

Reply:
Oh, yeesh! Yeah, I can't believe I missed that. And I'm somewhat surprised that the command runs happily without them. I'll fix right away.